### PR TITLE
BACK-1690: Adding Labels page

### PR DESCRIPTION
**Pull request by katerinachinnappan**
## Goal

* Adding a labels page to display non-paginated list of available labels. Only label names are displayed. 
* `Add Label` button does not function yet. This functionality will be implemented in another PR.

Tickets:

- [https://getpocket.atlassian.net/browse/BACK-1690](https://getpocket.atlassian.net/browse/BACK-1690)

Video

https://user-images.githubusercontent.com/16909783/204396720-8c70eb1a-1b66-4886-afbc-a32db03e6468.mov

### CODE REVIEW

**Review by nina-py — DISMISSED** [review]
Fantastic work! 🚀

File: src/collections/pages/LabelListPage/LabelListPage.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+          <Button
+            component={Link}
+            to="/collections/labels/add/"
+            buttonType="hollow"
+          >
+            Add label
+          </Button>
+        </Box>
+      </Box>
+      {console.log(data)}
```
Comment (jpetto) [inline src/collections/pages/LabelListPage/LabelListPage.tsx:30]: a little debug snuck in 😄

File: src/collections/pages/LabelListPage/LabelListPage.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  return (
+    <>
+      <Box display="flex">
+        <Box flexGrow={1} alignSelf="center">
+          <h1>Labels</h1>
+        </Box>
+        <Box alignSelf="center">
+          <Button
+            component={Link}
+            to="/collections/labels/add/"
```
Comment (jpetto) [inline src/collections/pages/LabelListPage/LabelListPage.tsx:23]: just to verify - this page doesn't exist yet, right?

**Review by jpetto — CHANGES_REQUESTED** [review]
one small question, and we should remove the debug statement, but otherwise i think this looks good code-wise.

File: src/collections/pages/LabelListPage/LabelListPage.tsx (hunk 2 above)
Comment (Herraj) [reply src/collections/pages/LabelListPage/LabelListPage.tsx:23]: Yes!

Comment (Herraj) [reply src/collections/pages/LabelListPage/LabelListPage.tsx:23]: Also, I think we have a custom button component that you can use and it comes with all the styles. Could be worth checking and making the code less redundant given our pending mui5 chore?

**Review by Herraj — APPROVED** [review]
Good stuff!

File: src/collections/pages/LabelListPage/LabelListPage.tsx (hunk 1 above)
Comment (katerinachinnappan) [reply src/collections/pages/LabelListPage/LabelListPage.tsx:30]: good catch!

File: src/collections/pages/LabelListPage/LabelListPage.tsx (hunk 2 above)
Comment (katerinachinnappan) [reply src/collections/pages/LabelListPage/LabelListPage.tsx:23]: @Herraj correct me if I am wrong but I think this is using the custom `Button` component from `_shared/components`?

Comment (Herraj) [reply src/collections/pages/LabelListPage/LabelListPage.tsx:23]: oh whoops, yes you are right. All good!

File: src/curated-corpus/components/actions/ScheduleCorpusItemAction/ScheduleCorpusItemAction.test.tsx [hunk 3]
```diff
@@ -44,7 +44,7 @@ describe('The ScheduleCorpusItemAction', () => {
   });
 
   it('completes the action successfully', async () => {
-    const today = DateTime.local();
+    const today = DateTime.local().plus({ days: 1 });
```
Comment (nina-py) [inline src/curated-corpus/components/actions/ScheduleCorpusItemAction/ScheduleCorpusItemAction.test.tsx:47]: Looks good! I was wondering if it's worth adding a comment here for posterity why this change was made

File: src/curated-corpus/helpers/helperFunctions.test.ts [hunk 4]
```diff
@@ -138,7 +138,7 @@ describe('helperFunctions ', () => {
     });
   });
 
-  describe('downloadAndUploadApprovedItemImageToS3 function', async () => {
+  describe('downloadAndUploadApprovedItemImageToS3 function', () => {
```
Comment (nina-py) [inline src/curated-corpus/helpers/helperFunctions.test.ts:141]: Nice!

File: src/curated-corpus/components/actions/ScheduleCorpusItemAction/ScheduleCorpusItemAction.test.tsx (hunk 3 above)
Comment (katerinachinnappan) [reply src/curated-corpus/components/actions/ScheduleCorpusItemAction/ScheduleCorpusItemAction.test.tsx:47]: will do !

**Review by nina-py — APPROVED** [review]

**Review by jpetto — APPROVED** [review]
🚀
